separate_curves.py

```diff
@@ -22,7 +22,7 @@
     "name": "Separate Curve Object",
     "author": "Agnieszka Pas",
     "version": (1, 1, 0),
-    "blender": (2, 79, 0),
+    "blender": (2, 80, 0),
     "location": "View3D > Tools Panel > Tools Tab > Edit",
```

Comment:
It should be also updated. Feel free to correct the path, or I can do it later (it will be no problem).